src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -335,7 +337,9 @@ const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
     const updatedAppDetails = {
       description: formState.values.appFullDescription,
       id: modalDetails.userAppID,
-      labels: formState.values.appLabels.split(' '),
+      labels: formState.values.appLabels.length
+        ? formState.values.appLabels.split(' ')
+        : [],
```

Comment:
This code is the same from lines 313-315. Moving it to a function would avoid repetition.